GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -107,13 +107,16 @@ extension ChatWrapperViewController: ChatControllerDelegate {
                 if let errorDescription = error.errorDescription {
                     Toast.show(message: errorDescription)
                 }
-                
+            case .conversationCreationError:
+                print("** Error: \(error.errorType.rawValue)")
+                if let errorDescription = error.errorDescription {
+                    Toast.show(message: errorDescription)
+                }
             case .failedToSendCustomAttributes:
                 print("** Error: \(error.errorType.rawValue)")
                 if let errorDescription = error.errorDescription {
                     Toast.show(message: errorDescription)
                 }
```

Comment:
Can we somehow merge these cases?

Reply:
Thanks for reviewing, you are right we can merge all the cases.
We've discussed it in the iOS team and we decided to keep them separate so it will be easier for us to differentiate between the cases.